nuspec/Frends.Csv.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
   <metadata>
     <id>Frends.Csv</id>
-    <version>1.0.0.0</version>
+    <version>1.0.5.0</version>
```

Comment:
Version should be 1.1.0, the third version number is automatically set by the build server